dev/com.ibm.ws.wsbytebuffer/resources/com/ibm/ws/bytebuffer/internal/resources/ByteBufferMessages.nlsprops

```diff
@@ -0,0 +1,40 @@
+###############################################################################
+# Copyright (c) 2010 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+# # {0} description of each insert field
+# MSG_DESCRIPTIVE_NAME_CWSJX0000=CWSJX0000I: This is a message with inserts {0}
+# MSG_DESCRIPTIVE_NAME_CWSJX0000.explanation=Explanation text for the message
+# MSG_DESCRIPTIVE_NAME_CWSJX0000.useraction=User action text for the message
+#
+#COMPONENTPREFIX CWWKO
+#COMPONENTNAMEFOR CWWKO Websphere kernel channel framework service: byte buffer support
+#CMVCPATHNAME com.ibm.ws.channelfw/resources/com/ibm/ws/bytebuffer/internal/resources/ByteBufferMessages.nlsprops
+#NLS_ENCODING=UNICODE
+#
+#ISMESSAGEFILE TRUE
+# NLS_MESSAGEFORMAT_VAR
+
+### WSBB will use message ids from 0600-0799
+
+NOT_VALID_CUSTOM_PROPERTY=CWWKO0600E: The custom property {0} has a value of {1}. This value is not valid.
+NOT_VALID_CUSTOM_PROPERTY.explanation=A custom property was specified with a value that is outside the range of valid values.
+NOT_VALID_CUSTOM_PROPERTY.useraction=Correct the configuration error.
+
+UNRECOGNIZED_CUSTOM_PROPERTY=CWWKO0601W: The custom property {0} specified for the WsByteBuffer component is not valid.
+UNRECOGNIZED_CUSTOM_PROPERTY.explanation=A custom property was specified incorrectly.
+UNRECOGNIZED_CUSTOM_PROPERTY.useraction=Correct the configuration error.
+
+CONFIG_VALUE_NUMBER_EXCEPTION=CWWKO0602E: The WsByteBuffer component generated a NumberFormatException exception when processing the following property: Property Name: {0}  Value: {1}
```

Comment:
Please change `NumberFormatException exception` to `NumberFormatException error` (replace "exception" with "error").

Unless the property and value names are long, maybe shorten `the following property: Property Name: {0} Value: {1}` to `the {0} property, which has a value of {1}`.